### PR TITLE
Fix/parse result

### LINKED ISSUE #34: Clarify the examples and parameters for each dbt task
### Issue description

# Problem
Currently, the same example is used for each task.

# To do
- provide clear documentation and examples for each task
- clarify which parameters are needed for each task e.g. `Deps` task doesn't have the `thread` parameter

### LINKED ISSUE #37: `dbt.cli.Deps` task returns an error `java.io.FileNotFoundException: /tmp/kestra-wd/tmp/xxx/target/run_results.json`
### Expected Behavior

reproducer:

```yaml
id: dbtGitDockerBigQuery
namespace: dev

tasks:
  - id: dbt
    type: io.kestra.core.tasks.flows.WorkingDirectory
    tasks:
    - id: cloneRepository
      type: io.kestra.plugin.git.Clone
      url: https://github.com/dbt-labs/jaffle_shop
      branch: main

    - id: dbt-deps
      type: io.kestra.plugin.dbt.cli.Deps
      runner: DOCKER
      dbtPath: /usr/local/bin/dbt
      dockerOptions:
        image: ghcr.io/kestra-io/dbt-bigquery:latest
      inputFiles:
        .profile/profiles.yml: |
          jaffle_shop:
            outputs:
              dev:
                type: bigquery
                dataset: dwh
                fixed_retries: 1
                keyfile: sa.json
                location: EU
                method: service-account
                priority: interactive
                project: geller
                threads: 8
                timeout_seconds: 300
            target: dev
        sa.json: "{{ secret('GCP_CREDS') }}"

    - id: dbt-build
      type: io.kestra.plugin.dbt.cli.Build
      runner: DOCKER
      dbtPath: /usr/local/bin/dbt
      dockerOptions:
        image: ghcr.io/kestra-io/dbt-bigquery:latest
      inputFiles:
        .profile/profiles.yml: |
          jaffle_shop:
            outputs:
              dev:
                type: bigquery
                dataset: dwh
                fixed_retries: 1
                keyfile: sa.json
                location: EU
                method: service-account
                priority: interactive
                project: geller
                threads: 8
                timeout_seconds: 300
            target: dev
        sa.json: "{{ secret('GCP_CREDS') }}"
```

### Actual Behaviour

image

full traceback:

```
dbt-deps
23s
Image pulled [ghcr.io/kestra-io/dbt-bigquery:latest]
Starting command with container id 161caeb3e1981f891f246decfe27861841c2e4cda383384adc9a33c1c8094c01 [/bin/sh -c /usr/local/bin/dbt --log-format json deps --profiles-dir /tmp/kestra-wd/tmp/3JmZq0jWaAuOBtMmcYWcMP/.profile]
[Date: 2023-07-17T23:11:20.477322Z] [Thread: MainThread] [Type: MainReportVersion] Running with dbt=1.5.2 
[Date: 2023-07-17T23:11:20.691006Z] [Thread: MainThread] [Type: DepsNoPackagesFound] Warning: No packages were found in packages.yml 
Command succeed with code 0
/tmp/kestra-wd/tmp/3JmZq0jWaAuOBtMmcYWcMP/target/run_results.json (No such file or directory)
java.io.FileNotFoundException: /tmp/kestra-wd/tmp/3JmZq0jWaAuOBtMmcYWcMP/target/run_results.json (No such file or directory)
	at java.base/java.io.FileInputStream.open0(Native Method)
	at java.base/java.io.FileInputStream.open(Unknown Source)
	at java.base/java.io.FileInputStream.<init>(Unknown Source)
	at com.fasterxml.jackson.core.TokenStreamFactory._fileInputStream(TokenStreamFactory.java:262)
	at com.fasterxml.jackson.core.JsonFactory.createParser(JsonFactory.java:1030)
	at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3542)
	at io.kestra.plugin.dbt.ResultParser.parseRunResult(ResultParser.java:38)
	at io.kestra.plugin.dbt.cli.AbstractDbt.run(AbstractDbt.java:100)
	at io.kestra.plugin.dbt.cli.AbstractDbt.run(AbstractDbt.java:20)
	at io.kestra.core.runners.Worker$WorkerThread.run(Worker.java:641)
```

### Steps To Reproduce

_No response_

### Environment Information

- Kestra Version: 0.10.1 develop full

### Example flow

_No response_

### PR DESCRIPTION
**Pull request by loicmathieu**
Fixes #37 
Fixes #34

### CODE REVIEW

**Review by anna-geller — APPROVED** [review]
LGTM, thanks so much!

File: src/main/java/io/kestra/plugin/dbt/cli/AbstractDbt.java [hunk 1]
```diff
@@ -56,7 +57,7 @@ public abstract class AbstractDbt extends AbstractBash implements RunnableTask<S
 
     @Builder.Default
     @Schema(
-        title = "The path to dbt cli"
+        title = "The path to the dbt cli"
```
Comment (anna-geller) [inline src/main/java/io/kestra/plugin/dbt/cli/AbstractDbt.java:60]:
```suggestion
        title = "The path to the dbt project directory"
```

might be more specific

File: src/main/java/io/kestra/plugin/dbt/cli/Setup.java [hunk 2]
```diff
@@ -32,6 +34,32 @@
     title = "Setup dbt",
     description = "Install pip package & init profile directory"
 )
+@Plugin(
+    examples = {
+        @Example(
+            full = true,
+            title = "Invoke dbt `build` command",
```
Comment (brian-mulier-p) [inline src/main/java/io/kestra/plugin/dbt/cli/Setup.java:41]: Looks incorrect as we already have a Build task ? :thinking:

**Review by brian-mulier-p — APPROVED** [review]
Small mistake otherwise LGTM

File: src/main/java/io/kestra/plugin/dbt/cli/Setup.java (hunk 2 above)
Comment (loicmathieu) [reply src/main/java/io/kestra/plugin/dbt/cli/Setup.java:41]: Yes :(  I double-checked all those copy/paste, but one has slipped through.

**Pull request merged**